### PR TITLE
enh(UI): Add a “Parent alias“ column  on the monitoring resources sta…

### PR DESCRIPTION
**Pull request by Jordan91**
Add a “Parent alias“ column  on the monitoring resources status

## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 22.04.x

### CODE REVIEW

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -15162,3 +15162,7 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
+# msgstr ""
+
+
```
Comment (Thebarda) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion

# msgid "Parent alias
# msgstr ""

```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16976,3 +16976,7 @@ msgstr "Veuillez utiliser l'interface Web pour installer Centreon."
 
 msgid "Please use Web UI to update Centreon."
 msgstr "Veuillez utiliser l'interface Web pour mettre à jour Centreon."
+# msgid "Parent alias
+# msgstr "Alias parent"
+
+
```
Comment (Thebarda) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion

msgid "Parent alias
msgstr "Alias parent"

```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -15613,3 +15613,7 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
+# msgstr ""
+
+
```
Comment (Thebarda) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion

# msgid "Parent alias
# msgstr ""

```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -15601,3 +15601,5 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
+# msgstr ""
```
Comment (Thebarda) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:15606]:
```suggestion

# msgid "Parent alias
# msgstr ""

```

**Review by smau-centreon — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -15162,3 +15162,7 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
```
Comment (jdelpierre) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
# msgid "Parent alias"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -16976,3 +16976,7 @@ msgstr "Veuillez utiliser l'interface Web pour installer Centreon."
 
 msgid "Please use Web UI to update Centreon."
 msgstr "Veuillez utiliser l'interface Web pour mettre à jour Centreon."
+# msgid "Parent alias
```
Comment (jdelpierre) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
# msgid "Parent alias"
```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -15613,3 +15613,7 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
```
Comment (jdelpierre) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
# msgid "Parent alias"
```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -15601,3 +15601,5 @@ msgstr ""
 
 #  msgid "Warning, maximum size exceeded for input '%s' (max: %d), it will be truncated upon saving"
 #  msgstr ""
+# msgid "Parent alias
```
Comment (jdelpierre) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:15605]:
```suggestion
# msgid "Parent alias"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -16976,3 +16976,7 @@ msgstr "Veuillez utiliser l'interface Web pour installer Centreon."
 
 msgid "Please use Web UI to update Centreon."
 msgstr "Veuillez utiliser l'interface Web pour mettre à jour Centreon."
+
+msgid "Parent alias
+msgstr "Alias parent"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Alias du parent"
```

**Comment by bdauria** [comment]
@Jordan91 you need to check why the build is failing

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by JKancel — APPROVED** [review]

**Pull request merged**